src/main/java/org/skriptlang/skript/bukkit/recipes/elements/CondSmithingNBT.java

```diff
@@ -0,0 +1,84 @@
+package org.skriptlang.skript.bukkit.recipes.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.Recipe;
+import org.bukkit.inventory.SmithingRecipe;
+import org.bukkit.inventory.SmithingTransformRecipe;
+import org.bukkit.inventory.SmithingTrimRecipe;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Recipe Allows NBT Copy")
+@Description({
+	"Checks whether the recipe allows nbt copying from the base item to the result item of the recipe.",
+	"NOTES:",
+	"This condition can only be used with Smithing Recipes on PaperMC version 1.19",
+	"This condition can be used with Smithing Transform and Smithing Trim Recipes on PaperMC version 1.20.0 -> 1.20.4",
+	"This condition is redundant for PaperMC versions 1.20.5+ due to the removal in favor of copying data components and will always return true"
```

Comment:
is there a reason for picking always true instead of always false or just not registering?
I'm also not sure why this isn't merged with data component condition, because I imagine most users consider components to be nbt.

Reply:
I could make it where it doesn't register. I figured I would keep it, so if users (below 1.20.5) use this and then switch to 1.20.5 or higher, it would just throw a warning instead of flat out erroring. 
Or I could put both the NBT and Data Eff's into 1 and same with Conds.
Up to you, let me know

Reply:
I think they should be merged. since they do the same thing functionally